README.md

```diff
@@ -98,6 +98,15 @@ Hurray! You successfully made a contribution!
 
 ---
 
+## <ins> What if I have a Conflict? </ins>
+
+#### A GitHub conflict is when people make changes to the same area or line in a file. This must be fixed before it is merged in order to prevent collision in the main branch. 
+
+* #### To read more about this, go to [Github Docs - About Merge Conflicts](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/about-merge-conflicts)
```

Comment:
```suggestion
* #### To read more about this, go to the [Github Docs - About Merge Conflicts](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/about-merge-conflicts) page.
```

Reply:
"The GitHub Docs" Ummm : / Actually (ig) it's gramatically fine